src/fastmcp/prompts/prompt.py

```diff
@@ -266,23 +258,13 @@ def from_function(
 
     def _convert_string_arguments(self, kwargs: dict[str, Any]) -> dict[str, Any]:
         """Convert string arguments to expected types based on function signature."""
-        from fastmcp.server.context import Context
-
         sig = inspect.signature(self.fn)
         converted_kwargs = {}
```

Comment:
For complete safety, this should be operating on the function `without_injected_parameters`, will change